dbos/_core.py

```diff
@@ -95,6 +95,7 @@
 F = TypeVar("F", bound=Callable[..., Any])
 
 TEMP_SEND_WF_NAME = "<temp>.temp_send_workflow"
+DEBOUNCER_WORKFLOW_NAME = "dbos_debouncer_workflow"
```

Comment:
Can we make this name something like "<DBOS>._debouncer_workflow" so it'll unlikely to clash with user functions? It'll also be easier for the console to filter out those workflows.